lib/barbeque_client/runner.rb

```diff
@@ -7,28 +7,63 @@ module BarbequeClient
   class Runner < Sinatra::Base
     set :port, ENV['PORT'] || 3003
 
+    class << self
+      # global variable
+      def messages
+        @messages ||= {}
+      end
+    end
+
     post '/v1/job_executions' do
```

Comment:
If this is for our in-house version, we can define this with a module in in-house plugin and include it from `Runner`. (Of course it is not related to this PR and it is not required here.)